tests/Resources/ti.ui.animation.test.js

```diff
@@ -0,0 +1,420 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020-Present by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+/* eslint no-undef: "off" */
+'use strict';
+
+const should = require('./utilities/assertions');
+
+describe('Titanium.UI.Animation', () => {
+	let instance;
+
+	beforeEach(() => {
+		// TODO: Create an instance of the type!
+		instance = Ti.UI.createAnimation();
+	});
+
+	afterEach(() => {
+		// TODO: Do any cleanup of the instance we need to do!
+		instance = null;
+	});
+
+	describe('properties', () => {
+		describe.android('.anchorPoint', () => {
+			it('is a Point', () => {
+				instance.anchorPoint = { x: 0, y: 0 };
+				should(instance).have.a.property('anchorPoint').which.is.an.Object();
+				should(instance.anchorPoint).have.properties(['x', 'y']);
```

Comment:
<!--
  0 failure: 
  2 warning:  tests/Resources/t..., tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.animation.test.js;
  Line: 32;
-->

- :warning: tests/Resources/ti.ui.animation.test.js line 32 – A space is required after '['. (array-bracket-spacing)
- :warning: tests/Resources/ti.ui.animation.test.js line 32 – A space is required before ']'. (array-bracket-spacing)